stdlib/2/_sre.pyi

```diff
@@ -21,6 +21,7 @@ class SRE_Match(object):
     def groups(self) -> Tuple[Optional[str], ...]: ...
     def span(self) -> Tuple[int, int]:
         raise IndexError()
+    def regs(self) -> Tuple[Tuple[int, int], ...]: ...  # undocumented
```

Comment:
This is actually a property, not a method:
```
In [141]: match=re.match(".", "sdfsdfj")                                                                                                                                           

In [142]: match                                                                                                                                                                    
Out[142]: <re.Match object; span=(0, 1), match='s'>

In [143]: match.regs                                                                                                                                                               
Out[143]: ((0, 1),)
```
Could you add `@property`? Sorry for missing this so far.

Reply:
Wow, I've done a pretty bad job here thus far. But re property, wouldn't it be just an instance attribute everywhere? What would make it a `@property` instead of that?

Reply:
This:
```
In [144]: match.regs = ()                                                                                                                                                                                             
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
~/Dropbox/code/taxonomy/taxonomy/db/constants.py in <module>
----> 1 match.regs = ()

AttributeError: attribute 'regs' of 're.Match' objects is not writable
```